src/components/PageMetadata.tsx

```diff
@@ -21,230 +21,88 @@ type PropMeta = {
+    { property: `og:description`, content: desc },
+    { property: `og:type`, content: `website` },
+    { name: `twitter:card`, content: `summary_large_image` },
+    { name: `twitter:creator`, content: author || siteTitle },
+    { name: `twitter:site`, content: author || siteTitle },
+    { name: `twitter:title`, content: fullTitle },
+    { name: `twitter:description`, content: desc },
+    { name: `twitter:image`, content: ogImageUrl },
+    { property: `og:url`, content: url },
+    { property: `og:image`, content: ogImageUrl },
+    { property: `og:site_name`, content: siteTitle },
+    { name: `docsearch:description`, content: desc },
+  ]
 
   return (
-    <Helmet
-      htmlAttributes={{ lang: language }}
-      title={title}
-      titleTemplate={`%s | ${siteTitle}`}
-      link={[{ rel: "canonical", key: canonical, href: canonical }]}
-      meta={[
-        {
-          name: `description`,
-          content: desc,
-        },
-        {
-          name: `image`,
-          content: site.siteMetadata.image,
-        },
-        {
-          property: `og:title`,
-          content: `${title} | ${siteTitle}`,
-        },
-        {
-          property: `og:description`,
-          content: desc,
-        },
-        {
-          property: `og:type`,
-          content: `website`,
-        },
-        {
-          name: `twitter:card`,
-          content: `summary_large_image`,
-        },
-        {
-          name: `twitter:creator`,
-          content: site.siteMetadata.author,
-        },
-        {
-          name: `twitter:site`,
-          content: site.siteMetadata.author,
-        },
-        {
-          name: `twitter:title`,
-          content: `${title} | ${siteTitle}`,
-        },
-        {
-          name: `twitter:description`,
-          content: desc,
-        },
-        {
-          name: `twitter:image`,
-          content: ogImageUrl,
-        },
-        {
-          property: `og:url`,
-          content: siteUrl,
-        },
-        {
-          property: `og:image`,
-          content: ogImageUrl,
-        },
-        {
-          property: `og:video`,
-          content: `https://www.youtube.com/channel/UCNOfzGXD_C9YMYmnefmPH0g`,
-        },
-        {
-          property: `og:site_name`,
-          content: `ethereum.org`,
-        },
-        {
-          name: `docsearch:description`,
-          content: desc,
-        },
-      ].concat(meta)}
-    >
-      <script type="application/ld+json">
-        {`
-        {
-          "@context": "https://schema.org",
-          "@type": "Organization",
-          "url": "https://ethereum.org",
-          "email": "press@ethereum.org",
-          "name": "Ethereum",
-          "logo": "https://ethereum.org/og-image.png"
-        }
-      `}
-      </script>
-    </Helmet>
+    <Head>
+      <title>{fullTitle}</title>
+      {metadata.map((data) => (
+        <meta key={(data as NameMeta).name || (data as PropMeta).property} {...data} />
```

Comment:
<img width="784" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/ba8dd05c-55d9-44c2-8f57-614ae39367c1">

@nhsz I went with this because not all items in the list will have `.property` available, since the array items are of a unionized type... either `name` or `property` is required but not both. Removing one of these will produce `undefined` keys for some items. Using `.content` also wouldn't work here since there are duplicates.